redis/src/cmd.rs

```diff
@@ -353,6 +353,48 @@ impl RedisWrite for Cmd {
 
         CmdBufferedArgGuard(self)
     }
+
+    #[cfg(feature = "bytes")]
+    fn bufmut_for_next_arg(&mut self, capacity: usize) -> impl bytes::BufMut + '_ {
+        self.data.reserve(capacity);
```

Comment:
The underlying vector might have excess capacity, so this should calculate exactly how much to reserve.

Reply:
> After calling `reserve`, capacity wil be greater than or equal to `self.len() + additional`. Does nothing if capacity is already sufficient.

So extra complexity on our side is not needed.